src/compositemark/common.ts

```diff
@@ -23,6 +31,31 @@ export type GenericCompositeMarkDef<T> = GenericMarkDef<T> &
     clip?: boolean;
   };
 
+export interface CompositeMarkTooltipSummary {
+  fieldPrefix: string;
+  titlePrefix: string;
+}
+
+export function getCompositeMarkTooltip(
```

Comment:
What is `tooltipSummaryMap` btw?  Why `-Map` suffix?   Why not `tooltipSummaries`?